src/flags.zig

```diff
@@ -333,7 +337,8 @@ pub const ByteUnit = enum(u64) {
 
 const ByteSizeParseError = error{
     ParseOverflow,
-    InvalidCharacter,
+    MissingDigits,
```

Comment:
It's great to handle  `InvalidUnit` separately!

Just a nit: rename `MissingDigits` with something prefixed with `Invalid*` to align with `InvalidUnit` ... for example, `InvalidSize` or `InvalidAmount`.

Also, I think the old message was more informative for this case `"{s}: expected a size, but found '{s}'"`